axelrod/fingerprint.py

```diff
@@ -0,0 +1,294 @@
+import axelrod as axl
+import numpy as np
+import matplotlib.pyplot as plt
+from axelrod.strategy_transformers import JossAnnTransformer, DualTransformer
+from axelrod.interaction_utils import compute_final_score_per_turn as cfspt
+from axelrod import on_windows
+from collections import namedtuple
+from tempfile import NamedTemporaryFile
+
+
+Point = namedtuple('Point', 'x y')
+
+
+def create_jossann(coordinate, probe):
```

Comment:
If they are specific to the Ashlock fingerprint, I've no objection to them being static methods within the AshlockFingerprint class (note: static methods, not class methods).